packages/contract-addresses/addresses.json

```diff
@@ -127,21 +123,20 @@
         "etherToken": "0x0b1ba0af832d7c05fd64161e0db78e85978e8082",
         "exchange": "0x48bacb9266a570d521063ef5dd96e61686dbe788",
         "assetProxyOwner": "0x0000000000000000000000000000000000000000",
-        "erc20BridgeProxy": "0x8ea76477cfaca8f7ea06477fd3c09a740ac6012a",
+        "erc20BridgeProxy": "0x038f9b392fb9a9676dbaddf78ea5fdbf6c7d9710",
         "zeroExGovernor": "0x0000000000000000000000000000000000000000",
-        "libTransactionDecoder": "0xb48e1b16829c7f5bd62b76cb878a6bb1c4625d7a",
-        "forwarder": "0x5d3ad3561a1235273cbcb4e82fce63a0073d19be",
+        "forwarder": "0xe704967449b57b2382b7fa482718748c13c63190",
         "orderValidator": "0x0000000000000000000000000000000000000000",
         "dutchAuction": "0x0000000000000000000000000000000000000000",
         "coordinatorRegistry": "0xaa86dda78e9434aca114b6676fc742a18d15a1cc",
         "coordinator": "0x4d3d5c850dd5bd9d6f4adda3dd039a3c8054ca29",
         "multiAssetProxy": "0xcfc18cec799fbd1793b5c43e773c98d4d61cc2db",
         "staticCallProxy": "0x6dfff22588be9b3ef8cf0ad6dc9b84796f9fb45f",
-        "devUtils": "0xa31e64ea55b9b6bbb9d6a676738e9a5b23149f84",
+        "devUtils": "0x74341e87b1c4db7d5ed95f92b37509f2525a7a90",
         "exchangeV2": "0x48bacb9266a570d521063ef5dd96e61686dbe788",
-        "zrxVault": "0x1941ff73d1154774d87521d2d0aaad5d19c8df60",
-        "staking": "0x0d8b0dd11f5d34ed41d556def5f841900d5b1c6b",
-        "stakingProxy": "0x38ef19fdf8e8415f18c307ed71967e19aac28ba1",
+        "zrxVault": "0xc4df27466183c0fe2a5924d6ea56e334deff146a",
```

Comment:
Why did these need to be re-deployed?

Reply:
These are ganache snapshot addresses. Every public library in `DevUtils` is a separate deployment now, so it moves the deployer's nonce so we end up with different addresses.